builder/googlecompute/config.go

```diff
@@ -92,6 +93,12 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 	if c.ImageDescription == "" {
 		c.ImageDescription = "Created by Packer"
 	}
+	// Possible values:
+	//   "MIGRATE"
+	//   "TERMINATE"
+	if c.OnHostMaintenance == "" {
+		c.OnHostMaintenance = "MIGRATE"
```

Comment:
I think since we know that there are only two possible values, doing something like

```golang
if !(c.OnHostMaintenance == "MIGRATE" || c.OnHostMaintenance == "TERMINATE") {
    errs = packer.MultiErrorAppend(errs,
    errors.New("on_host_maintenance must be one of MIGRATE or TERMINATE."))
}